src/rpc/server.cpp

```diff
@@ -252,7 +252,7 @@ UniValue stop(const UniValue& params, bool fHelp)
 /**
  * Call Table
  */
-static const CRPCCommand vRPCCommands[] =
+extern const CRPCCommand vRPCCommands[] =
```

Comment:
I believe its a style thing, it is optional but i like to mark an extern even at the definition to make it clearer to someone reading the code that it is global.